src/components/Menus/Sidebar/DownloadPanel/DownloadPanel.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import SidebarPanel from "../SidebarPanel";
+import { useTranslation } from "react-i18next";
+import DownloadButton from "../../../DownloadButton/DownloadButton";
+import DesignSystemButton from "../../../DesignSystemButton/DesignSystemButton";
+import DownloadIcon from "../../../../assets/icons/download.svg";
+import "../../../../assets/stylesheets/DownloadPanel.scss";
+
+export const DownloadPanel = () => {
+  const { t } = useTranslation();
+
+  return (
+    <SidebarPanel heading={t("downloadPanel.heading")}>
+      <div className="download-panel__content">
+        <div className="download-panel__subtitle">
+          {t("downloadPanel.subtitle")}
+        </div>
+        <p>{t("downloadPanel.logInHint")}</p>
+        <div className="download-panel__button-container">
+          <DesignSystemButton
+            className="btn btn--primary download-panel__button"
+            text={t("downloadPanel.logInButton")}
+            type="primary"
+            onClick={() => {
+              console.log("Log in clicked");
+            }}
+          />
+          <DesignSystemButton
+            className="btn btn--secondary download-panel__button"
+            text={t("downloadPanel.signUpButton")}
+            type="secondary"
```

Comment:
The `type` still isn't pulling in the correct styles from the design system but does enable behaviours (eg `onHover`)